sites/fast-website/src/app/components/fast-frame/fast-frame.template.ts

```diff
@@ -343,39 +345,44 @@ export const FastFrameTemplate = html<FastFrame>`
                     <div class="control-container">
                         <div class="control-container-column">
                             <fast-radio tabIndex="${x =>
-                                x.expanded ? "0" : "-1"}">Radio 1</fast-radio>
+                                !x.expanded && x.isResponsive
+                                    ? "-1"
+                                    : "0"}">Radio 1</fast-radio>
                             <fast-radio tabIndex="${x =>
-                                x.expanded ? "0" : "-1"}">Radio 2</fast-radio>
+                                !x.expanded && x.isResponsive
+                                    ? "-1"
+                                    : "0"}">Radio 2</fast-radio>
                         </div>
                         <div class="control-container-grid">
                             <fast-switch tabIndex="${x =>
-                                x.expanded ? "0" : "-1"}"></fast-switch>
+                                !x.expanded && x.isResponsive
+                                    ? "-1"
+                                    : "0"}"></fast-switch>
                             <p>Toggle</p>
                             <fast-checkbox tabIndex="${x =>
-                                x.expanded
-                                    ? "0"
-                                    : "-1"}" class="checkbox"></fast-checkbox>
+                                !x.expanded && x.isResponsive
+                                    ? "-1"
+                                    : "0"}" class="checkbox"></fast-checkbox>
                             <p class="checkbox-label">Checkbox</p>
                         </div>
                     </div>
                     <fast-text-field placeholder="Text field" tabIndex="${x =>
-                        x.expanded ? "0" : "-1"}"></fast-text-field>
+                        !x.expanded && x.isResponsive ? "-1" : "0"}"></fast-text-field>
                     <div class="control-container-2">
                         <fast-slider tabIndex="${x =>
-                            x.expanded ? "0" : "-1"}"></fast-slider>
-                        <fast-flipper aria-hidden="false" tabIndex="${x =>
-                            x.expanded ? "0" : "-1"}"></fast-flipper>
-                        <fast-flipper disabled tabIndex="${x =>
-                            x.expanded ? "0" : "-1"}"></fast-flipper>
+                            !x.expanded && x.isResponsive ? "-1" : "0"}"></fast-slider>
+                        <fast-flipper aria-label="Example 'flipper' button" tabIndex="${x =>
+                            !x.expanded && x.isResponsive ? "-1" : "0"}"></fast-flipper>
+                        <fast-flipper disabled aria-label="Example disabled 'flipper' button"></fast-flipper>
```

Comment:
Just an FYI that normally these are hidden from screen readers, so they don't end up being exposed and are out of the tabindex. We should consider if we want to expose these just for the sake of exposing them. I don't know that there is any value as they are not actionable.

I'm curious as to why we might want any of these non-actionable but interactive elements exposed here.

Reply:
good point. they don't do anything actionable but are only there to show a sited user a visual change. I will remove the floppers from the tab index. With that said, and out of curiosity, should we also remove the tab index from all of the example controls?

Reply:
I’d just drop the tab index from the flipper (and aria-label) for now...let’s get feedback from our external review on this all up.